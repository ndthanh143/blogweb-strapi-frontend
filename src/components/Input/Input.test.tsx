import React from 'react';
import renderer, { ReactTestRendererJSON } from 'react-test-renderer';
import { Input } from '.';

describe('Input component', () => {
  test('should render correctly', () => {
    const component = renderer.create(<Input type="text" defaultValue="type here..." />);
    let tree = component.toJSON() as ReactTestRendererJSON;
    expect(tree).toMatchSnapshot();
  });
});
