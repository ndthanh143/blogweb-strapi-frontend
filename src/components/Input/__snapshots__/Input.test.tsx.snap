// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component should render correctly 1`] = `
<div
  className="relative flex items-center"
>
  <input
    className="px-4 py-2 w-full bg-transparent dark:text-gray-200 border-2 rounded-lg dark:border-dark-mode"
    defaultValue="type here..."
    type="text"
  />
</div>
`;
